migration/sql-files/111-rename-fields-search-and-trigger.sql

```diff
@@ -0,0 +1,97 @@
+
+----------------------------------------------------------------------------------------------------
+--- Copied from 082-iteration-related-changed.sql
+----------------------------------------------------------------------------------------------------
+
+-- looks very similar to step 071 but here we replace the 
+-- `WHERE i.id::text = NEW.Fields->>'system_iteration'` comparison with 
```

Comment:
@jarifibrahim I bet before this was called `system.iteration`, right?

Reply:
Yes @kwk . That's correct.

Reply:
well, why not tell it here?

Reply:
Is this enough https://github.com/fabric8-services/fabric8-wit/pull/2342/commits/58d2ba93963fec193041c9f0cc36e95138aa3a22 ?

Reply:
Well, I thought to have this: 
```sql
-- `WHERE i.id::text = NEW.Fields->>'system.iteration'` comparison with
-- `WHERE i.id = (NEW.Fields->>'system_iteration')::uuid` to use the 
```

Reply:
The comment was out of date. I had copied over the comment from `082-iteration-related-changed.sql` file. I've removed it since it didn't make sense in this case https://github.com/fabric8-services/fabric8-wit/pull/2342/commits/a2d6d28e0a7850c47aaefac877c2be0876f63fc9